src/applications/disability-benefits/526EZ/config/form.js

```diff
@@ -663,12 +774,98 @@ const formConfig = {
                 `disabilities.${index}.view:uploadPrivateRecords`,
                 formData,
               ) === 'no';
-            return hasRecords && requestsRecords;
+
+            return !isProd && hasRecords && requestsRecords;
           },
           uiSchema: {
             disabilities: {
               items: {
-                'ui:description': authorizationToDisclose,
+                'ui:description': recordReleaseDescription,
+                'ui:title': disabilityNameTitle,
+                'view:limitedConsent': {
+                  'ui:options': {
+                    updateSchema: () => ({
+                      title: limitedConsentTitle,
+                    }),
+                  },
+                },
+                limitedConsent: {
+                  'ui:title': limitedConsentTextTitle,
+                  'ui:options': {
+                    expandUnder: 'view:limitedConsent',
+                    expandUnderCondition: true,
+                  },
+                  'ui:required': (formData, index) =>
+                    _.get(
+                      `disabilities.${index}.view:limitedConsent`,
+                      formData,
+                    ),
+                },
+                'view:privateRecordsChoiceHelp': {
+                  'ui:description': limitedConsentDescription,
+                },
+                providerFacility: {
+                  'ui:options': {
+                    itemName: 'Provider',
+                    viewField: PrivateProviderTreatmentView,
+                    hideTitle: true,
+                  },
+                  items: {
+                    providerFacilityName: {
+                      'ui:title': 'Name of private provider or hospital',
+                      'ui:errorMessages': {
+                        pattern:
+                          'Provider name must be less than 100 characters.',
+                      },
+                    },
+                    'ui:validations': [validateDate],
+                    treatmentDateRange: dateRangeUI(
+                      'Approximate date of first treatment',
+                      'Approximate date of last treatment',
+                      'End of treatment must be after start of treatment',
+                    ),
+                    providerFacilityAddress: Object.assign(
+                      addressUI('', false),
+                      {
+                        street: {
+                          'ui:title': 'Street',
+                          'ui:errorMessages': {
+                            pattern:
+                              'Street address must be less than 20 characters.',
```

Comment:
I went ahead and removed the `ui:errorMessages` sections since it isn't shown and it's handled automagically. Will double check with BA to see if content needs to be custom or if the magic works just fine.